docs/appendix/production-ready.md

```diff
@@ -0,0 +1,74 @@
+# What is production ready?
+
+## 12 factor app
+Twelve factor apps have declarative system requirements, are suitable for
+deployment in Docker containers, do not have differences between deployment and
+production, and can scale up without significant changes to tooling,
+architecture, or development practices.
```

Comment:
```suggestion
Twelve factor apps;
  - have declarative system requirements, 
  - are suitable for deployment in Docker containers,
  - do not have differences between development and production,
and can scale up without significant changes to tooling, architecture, or development practices.
```

Har ikke testet lokalt hvordan det ser ut, men ville rette på (det jeg tror var) en typo med "between ~deployment~development and production," samtidig som jeg foreslo noe jeg synes økte lesbarheten av en lang setning =)

Reply:
Gode forslag, committed